resources/zib/zib-TNMTumorClassification.xml

```diff
@@ -0,0 +1,617 @@
+      </binding>
+      <mapping>
+        <identity value="zib-tnmtumorclassification-v1.0-2020EN" />
+        <map value="NL-CM:4.27.8" />
+        <comment value="V_VenousInvasion" />
+      </mapping>
+    </element>
+    <element id="Observation.component:anatomicalStage">
+      <path value="Observation.component" />
+      <sliceName value="anatomicalStage" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:anatomicalStage.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="399390009" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:anatomicalStage.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="AnatomicalStage" />
+      <definition value="The stage of the disease is the grouping based on the anatomical extent of the disease." />
+      <alias value="AnatomischStadium" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.4.27.1--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-tnmtumorclassification-v1.0-2020EN" />
+        <map value="NL-CM:4.27.9" />
+        <comment value="AnatomicalStage" />
+      </mapping>
+    </element>
+    <element id="Observation.component:prognosticStage">
+      <path value="Observation.component" />
+      <sliceName value="prognosticStage" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:prognosticStage.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="399390009" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:prognosticStage.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="PrognosticStage" />
+      <definition value="The Prognostic Stage group of the disease. Next to the anatomic extent of the disease additional prognostic factors are included in the grouping." />
+      <alias value="PrognostischStadium" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.4.27.4--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-tnmtumorclassification-v1.0-2020EN" />
+        <map value="NL-CM:4.27.10" />
+        <comment value="PrognosticStage" />
+      </mapping>
+    </element>
+    <element id="Observation.component:tumorLocalization">
```

Comment:
The TumorLocalization container is part of the Abnormality container, and should thus not be mapped to a separate `.component`, especially taking the cardinalities into account. For instance, for the Anatomical Location(s) belonging to the DistantMetastasis it is fine to use a separate `.component`, as there is only one single M_DistantMetastasis. Here you could potentially have multiple Abnormality containers, each which their own distinct set of TumorLocalization concepts. By using separate slices, how are you able to reconstruct to which Abnormality a certain TumorLocalization belongs?